### ISSUE TITLE
I need help ...

### ISSUE THREAD

**Issue by yppdr** [GuillaumeHullin/Grandstream-HT813#1]
I just received my HT813 and the only one that I find on the internet talking about freebox is your git.
I just followed the tutorial with freepbx but I couldn't make it all work.

I have no led to light and on the status page the FSX and FXO remain in "Not Registered"

Any idea of the problem?

**Comment by GuillaumeHullin** [comment]
Do you see any error in Asterisk logs?

**Comment by GuillaumeHullin** [comment]
(Yes I felt alone also when trying to set up the HT813 ahaha)

**Comment by yppdr** [comment]
The only errors I have on Asterisk are when I try to call via Zoiper5 but I think my in and out rules for calls are not correct
The error in question :
![image](https://user-images.githubusercontent.com/1734599/140944105-83ffea26-ed04-4e80-aa2c-cf5784f76f26.png)

**Comment by GuillaumeHullin** [comment]
Ok first try to setup FXS port <--> analog phone... can you please check that:
1- your WAN is connected to the same network as your Asterisk
2- your FXS port is plugged to an analog phone

When you call the extension, those the analog phone ring?

**Comment by yppdr** [comment]
I just tested and I have no ringtone on the annalog phone

**Comment by GuillaumeHullin** [comment]
Ok. When you check the status of FXS port on the HT813... does it says Registered? (I guess still not).
For testing purpose, you could try to disable the firewall and the intrusion detection... maybe during your test your device got blacklisted.

**Comment by yppdr** [comment]
I just tested and still the same.., I think I really have trouble configuring both the HT813 but also the PBX

**Comment by GuillaumeHullin** [comment]
Yeah I understand. In order to get more debug data you can setup a remote syslog with this tool https://github.com/MaxBelkov/visualsyslog/tree/v1.6.4
In your HT813 you can setup your syslog server under the page Advanced Settings

**Comment by yppdr** [comment]
Indeed, this software is very practical ^^ And I can see all the errors encountered .. Example after a reboot of the HT813 : https://hastebin.kiwinetwork.xyz/jirofibomo.rb

**Comment by GuillaumeHullin** [comment]
Ok so my guess would be to first solve the FXS registration :) try to see if you manage to get an error on that.

**Comment by hoanghien1982** [comment]
I got the same problem as yppdr.
If i created an extension in freebpx, FXO is registered. However, if I just created trunk, then HT813 never register.
FreeBPX just reject registration from HT813
[2021-12-07 00:58:08] NOTICE[8127] res_pjsip/pjsip_distributor.c: Request 'REGISTER' from '"02838244895" <sip:02838244895@IP>' failed for 'IP:45848' (callid: 1853934365-5062-1@BA.HG.BB.BFI) - Failed to authenticate
syslog from HT813
Message: [00:0b:82:e6:d8:6a] [1.0.13.3]SIPStack(1)::cb_rcv4xx: Received 401 response for transaction 105(REGISTER)

**Comment by hoanghien1982** [comment]
Stop firewall IPS or whitelist your IP then HT813 can register as trunk now.

**Comment by GuillaumeHullin** [comment]
Ok great :) @yppdr is it solved for you too?

**Comment by hoanghien1982** [comment]
Everything is ok . However I got very special problem . My HT813 behind the NAT while FreeBPX is public IP address, after registering, server can not connect to HT813 because it used its IP private for header.
Other extensions (phones behind NAT) still show correct URI with its IP public.
Any idea?

**Comment by GuillaumeHullin** [comment]
I see... well you could check the option **NAT Traversal** ... in the FXS and FXO options. Please note also that my settings where meant for an internal use and so I did not setup encryption on the traffic... if your traffic is not encrypted and goes via internet... well I guess you understand what I mean. One thing you could maybe do... if you have a Linux server nearby your HT813 then maybe you could pull an SSH tunnel between this Linux and FreePBX.... check this out https://www.ssh.com/academy/ssh/tunneling/example
